readme.rst

```diff
@@ -31,16 +31,23 @@ Installation
 I use **Vundle** as the tool that helps me to install all the plugins I need
 in a very simple way.
 
-1. **Requirements SLE-12**
+1a. **Requirements for SLE-12**
 
 .. code-block:: bash
 
-   sudo zypper install python-devel curl git python-pip python-docutils
+   sudo zypper install vim python-dev curl git python-pip python-docutils
```

Comment:
I accidentally changed the suse command, but saw my mistake after sending the pull request. Just merge and then add "python-devel" command back